chain-impl-mockchain/src/block.rs

```diff
@@ -1,6 +1,6 @@
 //! Representation of the block in the mockchain.
-use crate::key::*;
-use crate::transaction::*;
+use crate::key::Hash;
+use crate::transaction::SignedTransaction;
```

Comment:
non necessary changes.

Reply:
Yeah, I just jumped on the bad style. @qnikst, please avoid using glob imports except for prelude modules that are specifically designed for that.

Reply:
@mzabaluev ok, will do.